spring-integration-file/src/main/java/org/springframework/integration/file/remote/handler/FileTransferringMessageHandler.java

```diff
@@ -91,13 +92,33 @@ public void setRemoteDirectoryExpression(Expression remoteDirectoryExpression) {
 
 	/**
 	 * Specify a remote directory path SpEL expression.
+	 * @param remoteDirectoryExpression the remote directory expression
+	 * @since 5.0
+	 * @see #setRemoteDirectoryExpression(Expression)
+	 */
+	public void setRemoteDirectoryExpressionString(String remoteDirectoryExpression) {
+		setRemoteDirectoryExpression(EXPRESSION_PARSER.parseExpression(remoteDirectoryExpression));
+	}
+
+	/**
+	 * Specify a remote directory path SpEL expression.
 	 * @param temporaryRemoteDirectoryExpression the temporary remote directory expression
 	 * @see RemoteFileTemplate#setTemporaryRemoteDirectoryExpression(Expression)
 	 */
 	public void setTemporaryRemoteDirectoryExpression(Expression temporaryRemoteDirectoryExpression) {
 		this.remoteFileTemplate.setTemporaryRemoteDirectoryExpression(temporaryRemoteDirectoryExpression);
 	}
 
+	/**
+	 * Specify a remote directory path SpEL expression.
+	 * @param temporaryRemoteDirectoryExpression the temporary remote directory expression
+	 * @since 5.0
+	 * @see #setTemporaryRemoteDirectoryExpression(Expression)
+	 */
+	public void setTemporaryRemoteDirectoryExpressionString(String temporaryRemoteDirectoryExpression) {
+		setTemporaryRemoteDirectoryExpression(EXPRESSION_PARSER.parseExpression(temporaryRemoteDirectoryExpression));
+	}
+
 	protected String getTemporaryFileSuffix() {
```

Comment:
We should do the same for the expressions for the `AbstractInboundFileSynchronizer`.